### DISCUSSION TITLE
Washed out picture when using Windows 11 HDR + Software Screen Capture

### DISCUSSION THREAD

**Discussion by kliszaj** [awawa-dev/HyperHDR#823] [Q&A] [answered]
I'm running an HyperHDR 20.0.0.0 instance on windows and using UDPdraw to send our frame data to a RPi driving my LEDs which is also runnign HyperHDR 20.0.0.0.  When Windows is in SDR mode, everything looks great.  As soon as I enable HDR mode in Windows, the colours are very washed out.  Running LUT calibration doesn't seem to help anything.

SDR Screenshot
![shot214012](https://github.com/awawa-dev/HyperHDR/assets/3270133/885928cf-755a-490b-ae93-cc8fa3c27be1)

HDR Screenshot
![shot213930](https://github.com/awawa-dev/HyperHDR/assets/3270133/5f65289c-ed32-401a-8626-b010a156bf2f)

Seems to be a regression of a previous bug: [https://github.com/awawa-dev/HyperHDR/discussions/515](https://github.com/awawa-dev/HyperHDR/discussions/515).

Would appreciate any help is someone has had any luck in resolving this

**Comment by awawa-dev** [comment]
It's interesting that you can apparently capture the screen when you have the DirectX software grabber in HyperHDR disabled in the second screenshot. So this screenshot is useless. Anyway check you logs if the hardware acceleration and HDR is enabled in logs, you may also try to adjust HDR brigtness correction. LUT table is not used when hardware acceleration is enabled but you can disable it and it cause the software grabber to fallback to standard mode (HYperHDR v19 and below).

On your second screenshot (blurred) it looks like it's zero.
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/2854ac0e-63d6-4490-954f-afa81260f107)

**Reply by kliszaj** [reply]
Sorry, I don't understand - how could I capture the screen if the grabber is disabled? The 2nd screenshot represents the washed out image that I also see from my LEDs.  Adjusting hardware acceleration and brightness correction has no effect.  Maybe this is my problem:

`2024-04-13T17:26:42.418Z [MEDIA_FOUNDATION:AUTO] setHdrToneMappingMode to: enable, but the LUT file is currently unloaded`

When I toggle on HDR, it seems that the LUT fails to load...

**Reply by awawa-dev** [reply]
You tell me: on the second screenshot you have software capturing for the instance disabled. You can even see it in this blurry screenshot. I don't analyze partial or cut-off logs.
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/aca1fcd2-8c1e-4721-b61f-7d07d9089efb)

**Comment by kliszaj** [comment]
Oh interesting - it looks like this page was not reflecting the state of the setting on the Remote Control tab.  Well even if on here, the issue still remains:
![shot200720](https://github.com/awawa-dev/HyperHDR/assets/3270133/07f05b54-03c8-4de9-b2ca-6e72635679d2)


I've attached a full log file 
[logs.txt](https://github.com/awawa-dev/HyperHDR/files/14968300/logs.txt)

**Reply by awawa-dev** [reply, marked as answer]
That's what I already talk about. For what reason did you disable it? 
```
Target SDR brightness is set to 0 nits. Disabling wide gamut.
```

**Reply by kliszaj** [reply]
Thanks for your help with troubleshooting!
